src/sentry/tasks/integrations/github/pr_comment.py

```diff
@@ -351,7 +351,8 @@ def safe_for_comment(
                 OPEN_PR_METRIC_BASE.format(key="api_error"),
                 tags={"type": "unknown_api_error", "code": e.code},
             )
-        raise e
+            logger.exception("github.open_pr_comment.unknown_api_error")
```

Comment:
if we do this, will we ever reattempt to comment on the PR since we've been rate limited?